app/views/offers/show.html.erb

```diff
@@ -1,20 +1,23 @@
-<div class="show-header">
-  <h1><%= @offer.topic %></h1>
-</div>
-<div class="img-container">
-  <img src="https://images.unsplash.com/photo-1507413245164-6160d8298b31?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80" alt="Biology picture">
-</div>
-<div class="show-bottom">
-  <div class="card-description">
-    <p><%= @offer.description %></p>
+<div class="container">
```

Comment:
Remove this line, and make the necessary accompanying changes (delete the closing tag too, and deindent the content in between)